src/app/static/src/app/components/projectListPage.vue

```diff
@@ -1,19 +1,23 @@
 <template>
-    <div class="container mt-5">
+     <div class="container mt-5">
         <div class="row">
             <div class="col-lg-6 offset-lg-3 col-10 offset-1">
                 <h1 class="h3">{{ welcomeText }}</h1>
                 <div v-if="projects.length > 0">
-                    <ul class="list-unstyled lead">
-                        <li v-for="project in projects">
-                            <drop-down :text="project.name">
-                                <a class="dropdown-item"
-                                   v-for="region in project.regions"
-                                   href="#"
-                                   @click="(event) => navigate(event, project, region)">
-                                    {{ region.name }}
+                    <ul class="list-group list-unstyled lead">
+                        <li v-for="(project, index) in projects" :key="index"
+                        class="list-group-item d-flex justify-content-between align-items-center">
```

Comment:
I'm not keen on the styling - can we remove `list-group` (line 7) and `list-group-item` (line 9)? But could then add `my-1` for a bit of spacing between the lines. Also I think `align-items-center` is not doing anything here and can be removed.

For the button itself, should really be a `<button>` tag rather than `<span>`, and in this case I think it makes sense to style it red to indicate functionality - how about 
`<button class="btn btn-sm btn-danger">`?